pkg/convert/testdata/programs/required_providers/pcl/main.pp

```diff
@@ -0,0 +1,12 @@
+package "different-name-null" {
```

Comment:
I think this need to be "null" for PCL as well. When the binder looks up package descriptors it does it based on this label, and the first part of the resource/invoke token. So if this is "different-name-null" nothing will ever use it.